app/src/main/java/com/example/game/activities/LoginActivity.java

```diff
@@ -20,14 +20,14 @@
 import com.parse.ParseUser;
 
 public class LoginActivity extends AppCompatActivity {
+    public static final String TAG = "LoginActivity";
```

Comment:
I think you probably only need this in this class so mark it as private

Reply:
makes sense!